### ISSUE TITLE
GenieAuthentication Broken, LoadError: Sessions not defined

### ISSUE THREAD

**Issue by jackn11** [GenieFramework/GenieAuthentication.jl#16]
Whenever I try to load the GenieAuthentication package, I get the following error. This error is currently preventing me from using the Genie.jl package.

```
Precompiling project...
  ✗ GenieAuthentication
  16 dependencies successfully precompiled in 34 seconds (52 already precompiled)
  1 dependency errored. To see a full report either run `import Pkg; Pkg.precompile()` or load the package

(@v1.7) pkg> precompile
Precompiling project...
  ✗ GenieAuthentication
  0 dependencies successfully precompiled in 7 seconds (68 already precompiled)

ERROR: The following 1 direct dependency failed to precompile:

GenieAuthentication [e115e502-7e3a-11e9-29b2-aba8be6c6778]

Failed to precompile GenieAuthentication [e115e502-7e3a-11e9-29b2-aba8be6c6778] to /home/jackn/.julia/compiled/v1.7/GenieAuthentication/jl_L1CAKT.
ERROR: LoadError: UndefVarError: Sessions not defined
Stacktrace:
 [1] getproperty(x::Module, f::Symbol)
   @ Base ./Base.jl:35
 [2] top-level scope
   @ ~/.julia/packages/GenieAuthentication/bJ9oW/src/GenieAuthentication.jl:14
 [3] include
   @ ./Base.jl:418 [inlined]
 [4] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)
   @ Base ./loading.jl:1318
 [5] top-level scope
   @ none:1
 [6] eval
   @ ./boot.jl:373 [inlined]
 [7] eval(x::Expr)
   @ Base.MainInclude ./client.jl:453
 [8] top-level scope
   @ none:1
in expression starting at /home/jackn/.julia/packages/GenieAuthentication/bJ9oW/src/GenieAuthentication.jl:1
```

**Comment by essenciary** [comment]
We're currently in process of updating most of the packages in the Genie ecosystem and until that finishes, some things won't work well. 

What is the output of `pkg> status` in this project?

**Comment by jackn11** [comment]
I have included the output below. Note, I have tested it on windows and linux, and this is on a fresh install of julia on linux. This bug with Genie appears in both environments.

Please let me know if there is a way that I can avoid this error in the meantime and what the timeline is to get Genie up and running.

```
(@v1.7) pkg> status
      Status `~/.julia/environments/v1.7/Project.toml`
  [c43c736e] Genie v5.0.0
  [e115e502] GenieAuthentication v1.2.0
```

**Comment by essenciary** [comment]
@jackn11 Thanks for the info. It's because GenieAuthentication 1.2 works with Genie 4. I have released yesterday v 1.2.1 of GenieAuthentication to limit the Genie compat to v4. For compat with Genie 5, we'll have GenieAuthentication v2 which is in process of being registered. 

So right now you have 2 options: 
1/ use GenieAuthentication v1.2.1 with Genie 4
2/ use GenieAuthentication#Geniev5 with Genie 5 (and update to GenieAuthentication@2 in a few days, once it's registered).

**Comment by jackn11** [comment]
Thank you! It seems like the problem has been solved and as such I will close the issue.
